addons/resource.language.en_gb/resources/strings.po

```diff
@@ -12833,24 +12843,24 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#24116"
-msgid "Default TV Service"
+msgid "Default \"TV\" service"
 msgstr ""
 
 #. Description of setting "Videos -> Subtitles -> Default TV Service" with label #24116
 #: system/settings/settings.xml
 msgctxt "#24117"
-msgid "Select service that will be used as default to search for TV Show subtitles."
+msgid "Select the service that will be used as default to search for TV show subtitles."
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#24118"
-msgid "Default Movie Service"
+msgid "Default \"Movie\" service"
```

Comment:
No if this is lowercase the TV entry that goes aside it is also lowercase, this is why it was done this way.

So Ill make one lowercase if tv is also lowercase.